src/lsdb/core/crossmatch/kdtree_match.py

```diff
@@ -17,25 +19,36 @@ class KdTreeCrossmatch(AbstractCrossmatchAlgorithm):
     extra_columns = pd.DataFrame({"_dist_arcsec": pd.Series(dtype=np.dtype("float64"))})
 
     # pylint: disable=unused-argument,arguments-differ
-    def validate(self, n_neighbors: int = 1, radius_arcsec: float = 1, require_right_margin=True, **kwargs):
+    def validate(
+        self,
+        n_neighbors: int = 1,
+        radius_arcsec: float = 1,
+        min_radius_arcsec: float = 0,
+        require_right_margin=True,
+        **kwargs,
+    ):
         super().validate()
         validate_radius(radius_arcsec)
+        if min_radius_arcsec < 0:
```

Comment:
`validate_radius` ensures that radii are positive but here the minimum radius might default to 0.

Reply:
ok but why less than 0?

Reply:
It's just a complementary check to warn users that this radius must be non-negative. I improved the error message.

Reply:
I still think this is a bit difficult to read, calling a separate validation function after already performing the validation. I think raising the error instead of calling validate radius is better here

Reply:
Yes, I agree! That's what I did really, for some reason this view of the code is outdated.